app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
```

Comment:
here's what [YuriiKindrat](https://github.com/YuriiKindrat) said me about comprehensions with functions, idk if it applies to generators also cause they look alike.

![NVIDIA_Share_GdqWWBPbTf](https://github.com/mate-academy/py-elves-and-dwarves/assets/13889031/f92746ea-5d02-48d1-a88f-941a5e0a3fbd)